homeassistant/components/linear_garage_door/light.py

```diff
@@ -0,0 +1,84 @@
+"""Linear garage door light."""
+
+from datetime import timedelta
+import math
+from typing import Any
+
+from linear_garage_door import Linear
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ColorMode, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LinearUpdateCoordinator
+from .entity import LinearEntity
+
+SCAN_INTERVAL = timedelta(seconds=60)
```

Comment:
Oh and you don't need this as this means the state is updated every 60 seconds, but since the state is updated through the coordinator, that doesn't make a lot of sense.